jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/TFRecordExecutor.scala

```diff
@@ -1,73 +1,63 @@
 package ml4ir.inference.tensorflow
 
 import com.google.protobuf.MessageLite
-import org.tensorflow.{SavedModelBundle, Tensor, Tensors}
+import org.tensorflow.ndarray.{ByteNdArray, NdArray, NdArrays, Shape, StdArrays}
+import org.tensorflow.ndarray.buffer.{ByteDataBuffer, DataBuffers}
+import org.tensorflow.types.{TFloat32, TString}
+import org.tensorflow.{SavedModelBundle, Tensor}
 
 /**
-  * @see <a href="https://www.tensorflow.org/api_docs/java/reference/org/tensorflow/SavedModelBundle">SavedModelBundle</a>
-  * Base class for performing the actual Tensorflow execution {@see TFRecordExecutor#apply}
-  *
-  * @param dirPath location on local disk of the Tensorflow { @code SavedModelBundle} to be used for inference
-  *                Directory likely looks like:
-  *                model_bundle/
-  *                ├── assets
-  *                ├── saved_model.pb
-  *                └── variables
-  *                    ├── variables.data-00000-of-00001
-  *                    ├── variables.data-00000-of-00002
-  *                    ├── variables.data-00001-of-00002
-  *                    └── variables.index
-  * @param config instantiated struct with the values from the feature_config.yaml relevant for inference
-  */
+ * @see <a href="https://www.tensorflow.org/api_docs/java/reference/org/tensorflow/SavedModelBundle">SavedModelBundle</a>
```

Comment:
Why do we need those changes?

Reply:
I think this was just an indentation change from the IDE